pkg/scheduler/server/internal/pool.go

```diff
@@ -182,7 +182,12 @@ func (p *Pool) getConn(meta *schedulerv1pb.JobMetadata) (*conn, error) {
 
 	nsPool, ok := p.nsPool[meta.GetNamespace()]
 	if !ok {
-		return nil, fmt.Errorf("no connections available for namespace: %s", meta.GetNamespace())
+		// no connections available for namespace
+		// TODO: add a dead-letter go routine and log once, but
+		// don't err and spam users if the job can't be sent back
+		log.Debugf("No daprd connections available to send triggered job back to for namespace: %s, %+v", meta.GetNamespace(), meta)
```

Comment:
Couldn't `meta` contain PII that is now being logged?

```suggestion
		log.Debugf("No daprd connections available to send triggered job back to for namespace: %s", meta.GetNamespace())
```